site/docs/v1.3.0-beta.2/upgrade-to-1.3.md

```diff
@@ -0,0 +1,63 @@
+# Upgrading to Velero 1.3
+
+## Prerequisites
+
+- Velero [v1.2][3] installed.
+
+If you're not yet running Velero v1.2, see the following:
+
+- [Upgrading to v1.1][1]
+- [Upgrading to v1.2][2]
+
+## Instructions
+
+1. Install the Velero v1.3 command-line interface (CLI) by following the [instructions here][0].
+
+    Verify that you've properly installed it by running:
+
+    ```bash
+    velero version --client-only
+    ```
+
+    You should see the following output:
+
+    ```bash
+    Client:
+        Version: v1.3.0
```

Comment:
For the beta, these would report `v1.3.0-beta.2`, wouldn't they? Are you trying to avoid having to submit a separate PR for the final version?

Reply:
It makes sense to have these say beta.2 for now; all will be updated when we release 1.3. Fixed.

Reply:
I did leave the master copy showing `v1.3.0` so I don't have to change that again; I only changed the beta.2 copy